src/System.Security.Cryptography.X509Certificates/tests/System.Security.Cryptography.X509Certificates.Tests.csproj

```diff
@@ -63,6 +63,9 @@
       <Link>CommonTest\System\PlatformDetection.cs</Link>
     </Compile>
   </ItemGroup>
+  <ItemGroup Condition="'$(TargetGroup)'==''">
+    <Compile Include="ChainTests.netstandard17.cs" />
```

Comment:
Why not the next section?  (The tests should run fine as netcoreapp1.1 (which really means 1.2))